### PR TITLE
Add ZF2015-10

### PR DESCRIPTION
**Pull request by paragonie-scott**

### CODE REVIEW

**Comment by Ocramius** [comment]
This should include zf >2.0.0,<2.4.9 as per http://framework.zend.com/blog/zend-framework-1-12-17-and-2-4-9-released.html

File: zendframework/zend-crypt/2015-11-23.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title:      Potential Information Disclosure in Zend\Crypt\PublicKey\Rsa\PublicKey
+link:       http://framework.zend.com/security/advisory/ZF2015-10
+cve:        CVE-2015-7503
+branches:
+    2.4.x:
+        time:     2015-11-23 14:30:00
+        versions: [<2.4.9]
+    2.5.x:
+        time:     2015-11-23 14:30:00
+        versions: [<2.5.2]
```
Comment (stof) [inline zendframework/zend-crypt/2015-11-23.yaml]: this needs a lower bound, otherwise 2.4.10 will also be mark as impacted

Comment (stof) [inline zendframework/zend-crypt/2015-11-23.yaml]: btw, this is why tests are failing

**Comment by paragonie-scott** [comment]
Updated and squashed the commits. :)

**Comment by Ocramius** [comment]
Still needs to cover `zendframework/zendframework` as well tho :-\

**Comment by paragonie-scott** [comment]
How would you recommend doing that?

File: zendframework/zend-crypt/2015-11-23.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+title:      Potential Information Disclosure in Zend\Crypt\PublicKey\Rsa\PublicKey
+link:       http://framework.zend.com/security/advisory/ZF2015-10
+cve:        CVE-2015-7503
+branches:
+    2.4.x:
+        time:     2015-11-23 14:30:00
+        versions: [>=2.4.0,<2.4.9]
```
Comment (stof) [inline zendframework/zend-crypt/2015-11-23.yaml]: according to @Ocramius's comment, the lower bound should be 2.0.0 here

**Comment by stof** [comment]
@paragonie-scott create a second advisory for the ``zendframework/zendframework`` package (with the same content except for the composer reference)

File: zendframework/zendframework/ZF2015-10.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+title:      Potential Information Disclosure in Zend\Crypt\PublicKey\Rsa\PublicKey
+link:       http://framework.zend.com/security/advisory/ZF2015-10
+cve:        CVE-2015-7503
+branches:
+    2.4.x:
+        time:     2015-11-23 14:30:00
+        versions: [>=2.0.0,<2.4.9]
+    2.5.x:
```
Comment (Ocramius) [inline zendframework/zendframework/ZF2015-10.yaml]: this key should be dropped (after 2.5.0, all packages are independent. Sadly, our LTS is 2.4.x)

**Comment by Ocramius** [comment]
:+1:

**Pull request merged**